app/helpers/article_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module ArticleHelper
+  VALID_STATUSES = {
```

Comment:
why here? constants, put in model

Reply:
Constants have been moved to the model. I thought it was a good idea to keep the models thin :)